aggregateur/issues.py

```diff
@@ -0,0 +1,55 @@
+import requests
+import yaml
+
+GITHUB_API = "https://api.github.com"
+REPO = "etalab/schema.data.gouv.fr"
+PHASES = ["construction", "investigation"]
+
+
+def prepare_issue(item):
+    return {
+        "title": item["title"],
+        "created_at": item["created_at"],
+        "url": item["html_url"],
+        "nb_comments": item["comments"],
+    }
```

Comment:
I think that's more than enough metadata to display on the web page later

Reply:
Maybe other labels? We can do w/o on a first version.

Reply:
Added an extra field to the YAML name, only with label friendly names. We could pick another structure later if we need